spot_rl_experiments/spot_rl/envs/lang_env.py

```diff
@@ -428,11 +428,10 @@ def step(self, *args, **kwargs):
         try:
             main(spot, args.use_mixer, config, args.output)
         finally:
-            spot.power_off()
```

Comment:
@jimmytyyang  what exactly is a Remote spot?

Reply:
Seems like Remote Spot is the core computer connected to the spot.
Remote spot does not have method called `shutdown()`